CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+- Sentry Swift framework (#2162)
```

Comment:
`h`: We need to point out here and also in the docs that from now on, the SDK has a dependency on Swift.